README.md

```diff
@@ -93,14 +92,14 @@ Please be aware that the native `source.yml` connection set up in the package wi
 To connect your multiple schema/database sources to the package models, follow the steps outlined in the [Union Data Defined Sources Configuration](https://github.com/fivetran/dbt_fivetran_utils/tree/releases/v0.4.latest#union_data-source) section of the Fivetran Utils documentation for the union_data macro. This will ensure a proper configuration and correct visualization of connections in the DAG.
 
 
-## Step 4: Define Cost & Usage Report source table
+## Step 4: Define Cost & Usage Report source table 
 By default, this package assumes your AWS Cost & Usage Report lives in a source table called `aws_cloud_cost_report`. In the very likely case that this is not what your table is called, configure the following variable in your `dbt_project.yml`.
 
 ```yml
 # dbt_project.yml
 
 vars:
-    aws_cloud_cost_report_identifier: your_table_name 
+  aws_cloud_cost_report_identifier: your_table_name 
```

Comment:
This is a great feature, but can you explain why it is likely the table will be named something else? Are customers able to control the name? This is something we will need to inform quickstart about as this will complicate how the package will work as a quickstart data model.

Reply:
That would be because there's no default name, users HAVE to input a custom name in the connector setup form 
![image](https://github.com/user-attachments/assets/36f3918b-182a-46bf-ad8d-66c7b2a04f42)
This custom name appears to be used for both the connector schema _and_ table name (so the above data gets stored in `aws_cost_frank_test_hourly. aws_cost_frank_test_hourly`), but I'm confirming this with Frank. Regardless, I think it would be good to keep separate variables just in case, as our internal dataset (which doesn't use the Fivetran connector) has a schema name that's different from the report name


Quickstart should be able to grab the connector name right?

Reply:
the fact that there's no standard table name is really complicating union_data... the customers we've spoken with have _really_ emphasized wanting the union ability, so i'm going rogue locally and testing out a custom union_data macro that uses a dictionary like this as input:
```yml
  aws_cloud_cost_sources:
    - database: <db name>
      schema: <schema name>
      table: <table name>
      source: "{{ source('aws_cloud_cost', 'report') }}" # (optional) -- they can define their own sources and just tell use directly what they are 
    - database: ...
      schema: ...
```